mono/metadata/sre.c

```diff
@@ -2481,24 +2521,23 @@ reflection_setup_internal_class (MonoReflectionTypeBuilderHandle ref_tb, MonoErr
 }
 
 /**
- * reflection_create_generic_class:
+ * reflection_inflate_generic_class:
  * @tb: a TypeBuilder object
  * @error: set on error
  *
  * Creates the generic class after all generic parameters have been added.
  * On success returns TRUE, on failure returns FALSE and sets @error.
+ *
+ * This assumes that reflection_setup_internal_class has already set up
+ * ref_tb
  */
 static gboolean
-reflection_create_generic_class (MonoReflectionTypeBuilderHandle ref_tb, MonoError *error)
+reflection_inflate_generic_class (MonoReflectionTypeBuilderHandle ref_tb, MonoError *error)
```

Comment:
I don't think "inflate" is a great name for this.  I think the runtime usually uses "inflate" to mean "instantiate", but this function isn't creating a generic instance, it's creating the generic context for a generic type definition.